src/widgets/splits/Split.cpp

```diff
@@ -309,6 +330,41 @@ void Split::updateInputPlaceholder()
     this->input_->ui_.textEdit->setPlaceholderText(placeholderText);
 }
 
+void Split::joinChannelInNewTab(ChannelPtr channel)
+{
+    // Code opening a new tab was taken from:
+    // src/widgets/dialogs/switcher/NewTabItem.cpp#22 - NewTabItem::action
```

Comment:
Useful comment for me now while reviewing it, but unnecessary to keep in the code